cloudwatch/src/lambda_function.py

```diff
@@ -76,6 +76,16 @@ def _parse_cloudwatch_log(log, aws_logs_data, log_type):
     except (KeyError, ValueError):
         pass
 
+    # If ENRICH has value, add the properties
+    try:
+        if os.environ['ENRICH']:
+            properties_to_enrich = os.environ['ENRICH'].split(";")
+            for property_to_enrich in properties_to_enrich:
+                property_key_value = property_to_enrich.split("=")
+                log[property_key_value[0]] = property_key_value[1]
+    except (KeyError, ValueError):
+        pass
+
```

Comment:
Lets put this in `_enrich_logs_data`, that way we can do it once in an event and not every log line